tiledb/bioimg/converters/ome_tiff.py

```diff
@@ -270,20 +282,112 @@ def original_metadata(self) -> Dict[str, Any]:
 
 
 class OMETiffWriter(ImageWriter):
-    def __init__(self, output_path: str):
+    def __init__(self, output_path: str, ome: bool = True):
         self._output_path = output_path
+        self._ome = ome
 
     def write_group_metadata(self, metadata: Mapping[str, Any]) -> None:
-        tiffwriter_kwargs = json.loads(metadata["json_tiffwriter_kwargs"])
-        tiffwriter_kwargs.pop("append")
         self._writer = tifffile.TiffWriter(
-            self._output_path, shaped=False, append=False, **tiffwriter_kwargs
+            self._output_path, shaped=False, bigtiff=True, append=False, ome=self._ome
```

Comment:
Why don't we leave this as is and hardcode the `group_metadata` : 
```python
@property
    def group_metadata(self) -> Dict[str, Any]:
        writer_kwargs = dict(
            bigtiff=self._tiff.is_bigtiff,
            byteorder=self._tiff.byteorder,
            append=self._tiff.is_appendable,
            imagej=self._tiff.is_imagej,
            ome=self._tiff.is_ome,
        )
        return {"json_tiffwriter_kwargs": json.dumps(writer_kwargs)}
```

and after that as it was `self._writer = tifffile.TiffWriter(self._output_path, shaped=False, **tiffwriter_kwargs` calling that will do the job

Reply:
I am more inclined to store the actual tiff metadata and choose not to use them during export rather than storing a set of "ideal metadata." Having the metadata can provide debug info in case of erroneous ingestion. Also, ingestion and export are being decoupled since no OS/platform specific options (like byteorder) are shared